app/main.py

```diff
@@ -2,0 +1,18 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
Code Style: It's a good practice to use type annotations for all function parameters and return types. This improves code readability and can help with development tools that perform type checking, autocompletion, etc. In this case, the `cafe` parameter is missing a type annotation. It should be annotated with the `Cafe` type:

```python
def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```
This way, anyone reading your code can easily understand the expected type of `cafe`.

Reply:
I already use this structure